.github/workflows/ci.yaml

```diff
@@ -42,6 +42,7 @@ jobs:
     needs:
     - lint-chart
     strategy:
+      fail-fast: false
```

Comment:
Could we remove this?

Reply:
the pipeline is failing with no apparent reason without it

Reply:
ok, let's go with this fix, and potentially further improvement of the CI pipeline, like an upgrade of the k8s version, may help.